src/svc/firebase.rs

```diff
@@ -22,7 +22,7 @@ impl<'a> Hub<'a> {
         where for<'de> D: 'static + Send + Deserialize<'de>
     {
         let uri = Uri::from_str(&format!("https://{}.firebaseio.com/{}.json?shallow={}",
-                                         self.project_id(),
+                                         self.firebase_project_id(),
```

Comment:
Could we just make this a function argument instead? One service account may need to access multiple firebase projects...